troubleshooting/tools/tracevol.py

```diff
@@ -114,44 +140,53 @@ def format_table(arg, pvc_data, table):
         table.add_row([pvcname, "", "", False,
                        False, False])
         return
-    pool_name = get_pool_name(arg, volume_name)
+    pool_name = get_pool_name(arg, volume_name, isRBD)
     if pool_name == "":
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
     # get image id
+    # for cephfs, image_id means subvolume id
     image_id = get_image_uuid(volume_name)
     if image_id is None:
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
     # check image details present rados omap
-    pv_present, uuid_present = validate_volume_in_rados(
-        arg, image_id, pvname, pool_name)
-    image_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    # for cephfs its subvolume details
+    pv_present, uuid_present = validate_volume_in_rados(arg, image_id, pvname,
+                                                        pool_name, isRBD)
+    present_in_cluster = False
+    if isRBD == True:
```

Comment:
In python values are truthy or not. Just do `if isRBD;`, especially since this is a boolean.